src/category_theory/limits/shapes/zero.lean

```diff
@@ -163,20 +183,12 @@ section
 variable [has_zero_morphisms.{v} C]
 
 /--  An arrow ending in the zero object is zero -/
-@[ext]
 lemma zero_of_to_zero {X : C} (f : X ⟶ 0) : f = 0 :=
```

Comment:
```suggestion
@[simp]
lemma zero_of_to_zero {X : C} (f : X ⟶ 0) : f = 0 :=
```

Reply:
Oh, I forgot until the linter reminded me. This can't be a `@[simp]` lemma because the left hand side would be a metavariable. I've added a comment instead.

Reply:
Thanks for the reminder (-;